runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -57,6 +57,12 @@
             "append-checkpoint";
     private Timer appendCheckpointTimer = metricRegistry.timer(CHECKPOINT_TIMER_NAME);
 
+    /**
+     * Marker to insert to start a snapshot transaction to bypass sequencer token regression on sequencer failover.
+     * Persisting this marker freezes all tokens dispatched before this with the current/old sequencer.
```

Comment:
Maybe wrap the above two comment lines? They look a little too long compared to other lines in this file.